Windows build for 2.5.0?

Hey folks,

Could someone make a Windows build of [version 2.5.0](https://github.com/pcb2gcode/pcb2gcode/releases/tag/v2.5.0)? 

The README.md file reads:
> Windows prebuilt binaries (with all the required DLLs) are available in the [release](https://github.com/pcb2gcode/pcb2gcode/releases) page.

If Windows binaries will no longer be released, then this text should be adjusted.

Here's some feedback on the current README:

> You can easily build pcb2gcode for Windows with MSYS2 (http://sourceforge.net/projects/msys2/).

I'd change the link to https://www.msys2.org as that's a much friendlier website, and its' a lot clearer where to go and what to do.

> then run "MinGW-w64 Win32 Shell"

That's called "MSYS MINGW32" (for i686) or "MSYS MINGW64" (for x86) nowadays. The README could also use a warning that it's important which shell you start, as it seems to influence which compilers etc. are available to you.

> The following commands are for the i686 binary

I think nowadays it's better to list the commands for a 64-bit binary, and leave adjustments for a 32-bit as a side-note.

> Now let's download, build and install gerbv (version 2.6.1 is broken, don't use it)

Well, 2.6.0 is also broken:
```
Making all in src
make[2]: Entering directory '/home/Sybren/workspace/pcb2gcode/gerbv-2.6.0/src'
/bin/sh ../libtool --tag=CC   --mode=link gcc -DBACKEND_DIR='"/mingw64/share/gerbv/scheme/"' -DSCMSUBDIR='"scheme"' -mms-bitfields -mwindows -Wall  -I/mingw64/include/gtk-2.0 -I/mingw64/lib/gtk-2.0/include -I/mingw64/include/atk-1.0 -I/mingw64/include/pango-1.0 -mms-bitfields -I/mingw64/include/gdk-pixbuf-2.0 -I/mingw64/include/cairo -I/mingw64/include/harfbuzz -I/mingw64/include/freetype2 -I/mingw64/include/glib-2.0 -I/mingw64/lib/glib-2.0/include -DLIBDEFLATE_DLL -I/mingw64/include/webp -I/mingw64/include/pixman-1 -I/mingw64/include/fribidi -I/mingw64/include/libpng16 -I/mingw64/include/cairo -I/mingw64/include/freetype2 -I/mingw64/include/pixman-1 -I/mingw64/include/glib-2.0 -I/mingw64/lib/glib-2.0/include -I/mingw64/include/harfbuzz -I/mingw64/include/libpng16   -o gerbv.exe dynload.o attribute.o lrealpath.o main.o project.o scheme.o callbacks.o interface.o render.o libgerbv.la gerbv_icon.o -lm   -lmsimg32 -lgdk-win32-2.0 -lgdi32 -limm32 -lshell32 -lole32 -Wl,-luuid -latk-1.0 -lharfbuzz -lpangocairo-1.0 -lgtk-win32-2.0 -lpangowin32-1.0 -lgio-2.0 -lpango-1.0 -lgdk_pixbuf-2.0 -lcairo -lglib-2.0 -lintl -lgobject-2.0 -lcairo
libtool: link: gcc -DBACKEND_DIR=\"/mingw64/share/gerbv/scheme/\" -DSCMSUBDIR=\"scheme\" -mms-bitfields -mwindows -Wall -I/mingw64/include/gtk-2.0 -I/mingw64/lib/gtk-2.0/include -I/mingw64/include/atk-1.0 -I/mingw64/include/pango-1.0 -mms-bitfields -I/mingw64/include/gdk-pixbuf-2.0 -I/mingw64/include/cairo -I/mingw64/include/harfbuzz -I/mingw64/include/freetype2 -I/mingw64/include/glib-2.0 -I/mingw64/lib/glib-2.0/include -DLIBDEFLATE_DLL -I/mingw64/include/webp -I/mingw64/include/pixman-1 -I/mingw64/include/fribidi -I/mingw64/include/libpng16 -I/mingw64/include/cairo -I/mingw64/include/freetype2 -I/mingw64/include/pixman-1 -I/mingw64/include/glib-2.0 -I/mingw64/lib/glib-2.0/include -I/mingw64/include/harfbuzz -I/mingw64/include/libpng16 -o .libs/gerbv.exe dynload.o attribute.o lrealpath.o main.o project.o scheme.o callbacks.o interface.o render.o gerbv_icon.o -Wl,-luuid  ./.libs/libgerbv.dll.a -lmsimg32 -lgdk-win32-2.0 -lgdi32 -limm32 -lshell32 -lole32 -latk-1.0 -lharfbuzz -lpangocairo-1.0 -lgtk-win32-2.0 -lpangowin32-1.0 -lgio-2.0 -lpango-1.0 -lgdk_pixbuf-2.0 -lglib-2.0 -lintl -lgobject-2.0 -lcairo -L/mingw64/lib
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: project.o:project.c:(.bss+0x0): multiple definition of `screenRenderInfo'; mai
n.o:main.c:(.bss+0x20): first defined here
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: callbacks.o:callbacks.c:(.bss+0x0): multiple definition of `CALLBACKS_SAVE_FIL
E_TYPE'; main.o:main.c:(.bss+0x0): first defined here
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: callbacks.o:callbacks.c:(.bss+0x20): multiple definition of `screenRenderInfo'
; main.o:main.c:(.bss+0x20): first defined here
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: interface.o:interface.c:(.bss+0x0): multiple definition of `CALLBACKS_SAVE_FIL
E_TYPE'; main.o:main.c:(.bss+0x0): first defined here
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: interface.o:interface.c:(.bss+0x20): multiple definition of `screenRenderInfo'
; main.o:main.c:(.bss+0x20): first defined here
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: render.o:render.c:(.bss+0x0): multiple definition of `CALLBACKS_SAVE_FILE_TYPE
'; main.o:main.c:(.bss+0x0): first defined here
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: render.o:render.c:(.bss+0x20): multiple definition of `screenRenderInfo'; main
.o:main.c:(.bss+0x20): first defined here
collect2.exe: error: ld returned 1 exit status
make[2]: *** [Makefile:437: gerbv.exe] Error 1
make[2]: Leaving directory '/home/Sybren/workspace/pcb2gcode/gerbv-2.6.0/src'
make[1]: *** [Makefile:319: all-recursive] Error 1
make[1]: Leaving directory '/home/Sybren/workspace/pcb2gcode/gerbv-2.6.0'
make: *** [Makefile:248: all] Error 2
```
Gerb 2.6.0 is also outdated, it's so old that it predates the releases on their [releases page](https://github.com/gerbv/gerbv/releases). Now that I read more, it turns out that that's a fork because the original project stalled. From their README:

> While Gerbv is great software, the development on Source Force has stalled since many years with patches accumulating in the tracker and mailing list.

Their fork builds after running `pacman -S libtool mingw-w64-x86_64-autotools` and `./autogen.sh`.

Fortunately, pcb2gcode seems to build just fine with that newer version of Gerb.

> The required DLLs are:

You can automate this, so that people trying to follow the steps don't have to manually go over each library:

```
cp $(ldd pcb2gcode.exe  | cut -d'>' -f2 | cut -d'(' -f1 | grep mingw64 ) libs -v
```

And now I have an executable that works, at least on my machine:

```
C:\msys64\opt\pcb2gcode\bin>pcb2gcode.exe --version
2.5.0
Git commit: latest
Boost: 108100
Gerbv: 2.9.6~77211d
Geos: Not installed
```

So yeah, "You can easily build pcb2gcode for Windows with MSYS2" was not exactly true. But with some creativity, it was doable.

--- a/README.md
+++ b/README.md
@@ -41,7 +41,7 @@ There are pcb2gcode packages in the official repositories. You can install the w
 Unfortunately, these packages are seriously outdated. If you want to download the latest development version, go to "Installation from GIT".
 
 #### Windows
-Windows prebuilt binaries (with all the required DLLs) are available in the [release](https://github.com/pcb2gcode/pcb2gcode/releases) page.
+Windows prebuilt binaries (with all the required DLLs) may be available in the [release](https://github.com/pcb2gcode/pcb2gcode/releases) page. Due to difficulties with the CI pipeline, these are not reliably built and may be missing for certain releases. See below for instructions how to build pcb2gcode yourself.
 
 #### Mac OS X
 pcb2gcode is available in [Homebrew](http://brew.sh/). To install it open the "Terminal" app and run the following commands; pcb2gcode and the required dependencies will be automatically downloaded and installed:
@@ -101,23 +101,26 @@ Then follow the [common build steps](#commonbuild)
     $ sudo make install
 
 ### Windows
-You can easily build pcb2gcode for Windows with MSYS2 (http://sourceforge.net/projects/msys2/).
-Download MSYS2 and install it somewhere, then run "MinGW-w64 Win32 Shell" (if you want a i686 binary) or "MinGW-w64 Win64 Shell" (if you want a x86_64 binary). The following commands are for the i686 binary, if you want the x86_64 binary replace all the "/mingw32" with "/mingw64" and all the mingw-w64-i686-* packages with mingw-w64-x86_64-*
+You can build pcb2gcode for Windows with MSYS2 (https://www.msys2.org/).
+Download MSYS2 and install it somewhere, then run "MSYS MINGW64" (if you want a 64-bit x86_64 binary) or "MSYS MINGW32" (if you want a 32-bit i686 binary).
+
+The following commands are for the x86_64 binary, if you want the i686 binary replace all the "/mingw64" with "/mingw32" and all the mingw-w64-x86_64-* packages with mingw-w64-i686-*
 
     $ pacman -Sy
     $ pacman --needed -S bash pacman pacman-mirrors msys2-runtime
 
 Close and reopen the shell
 
-    $ pacman -Su
-    $ pacman --needed -S base-devel git mingw-w64-i686-gcc mingw-w64-i686-boost mingw-w64-i686-gtkmm
+    $ pacman -Su  # This may also close the shell; if so, reopen it.
+    $ pacman --needed -S base-devel git mingw-w64-x86_64-gcc mingw-w64-x86_64-boost mingw-w64-x86_64-gtkmm libtool mingw-w64-x86_64-autotool
 
-Now let's download, build and install gerbv (version 2.6.1 is broken, don't use it)
+Now let's download, build and install gerbv:
 
-    $ wget downloads.sourceforge.net/gerbv/gerbv-2.6.0.tar.gz
-    $ tar -xzf gerbv-2.6.0.tar.gz
-    $ cd gerbv-2.6.0/    
-    $ ./configure --prefix=/mingw32 --disable-update-desktop-database
+    $ git clone https://github.com/gerbv/gerbv.git
+    $ cd gerbv
+    $ git checkout v2.9.6  # Optional step. This version is known to work with pcb2gcode 2.5.0.
+    $ ./autogen.sh
+    $ ./configure --disable-update-desktop-database
     $ make
     $ make install
 
@@ -127,11 +130,19 @@ Finally, download and build pcb2gcode
     $ git clone https://github.com/pcb2gcode/pcb2gcode.git
     $ cd pcb2gcode/
     $ autoreconf -fvi
-    $ ./configure --prefix=/mingw32
+    $ ./configure
     $ make LDFLAGS='-s'
 
 The dynamically linked binary is &lt;msys2 installation folder&gt;/home/&lt;user&gt;/pcb2gcode/.libs/pcb2gcode.exe.
-You can find all the DLLs in &lt;msys2 installation folder&gt;/mingw32/bin; copy them in the same folder of pcb2gcode. The required DLLs are:
+You can find all the DLLs in &lt;msys2 installation folder&gt;/mingw32/bin; copy them in the same folder of pcb2gcode.
+These steps can be automated with:
+
+    $ mkdir my-pcb2gcode-build
+    $ cp .libs/pcb2gcode.exe my-pcb2gcode-build
+    $ cd my-pcb2gcode-build
+    $ cp $(ldd pcb2gcode.exe  | cut -d'>' -f2 | cut -d'(' -f1 | grep mingw64 | sort -u) .
+
+The required DLLs are:
  * libatk-1.0-0.dll
  * libboost_program_options-mt.dll
  * libbz2-1.dll

